python_http_client/exceptions.py

```diff
@@ -0,0 +1,53 @@
+class HTTPError(Exception):
+	''' Base of all other errors'''
+	def __init__(self,error):
+		self.code = error.code
+		self.reason = error.reason
+		#self.headers = error.headers
```

Comment:
Ah! Since it was not mentioned in the documentation, I missed it. Thanks for the tip! :)